README.md

```diff
@@ -1,2 +1,5 @@
 # git-playground
 following a bunch of git learning resources 
```

Comment:
please elaborte more on the purpose of the repo

Reply:
ok man i will do as indicated